bpfman-operator/test/integration/kprobe_test.go

```diff
@@ -0,0 +1,68 @@
+//go:build integration_tests
+// +build integration_tests
+
+package integration
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"regexp"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/kong/kubernetes-testing-framework/pkg/clusters"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	kprobeGoCounterKustomize       = "../../../examples/config/default/go-kprobe-counter"
+	kprobeGoCounterUserspaceNs     = "go-kprobe-counter"
+	kprobeGoCounterUserspaceDsName = "go-kprobe-counter-ds"
+)
+
+func TestKprobeGoCounter(t *testing.T) {
+	t.Log("deploying kprobe counter program")
+	require.NoError(t, clusters.KustomizeDeployForCluster(ctx, env.Cluster(), kprobeGoCounterKustomize))
+	addCleanup(func(context.Context) error {
+		cleanupLog("cleaning up kprobe counter program")
+		return clusters.KustomizeDeleteForCluster(ctx, env.Cluster(), kprobeGoCounterKustomize)
+	})
+
+	t.Log("waiting for go kprobe counter userspace daemon to be available")
+	require.Eventually(t, func() bool {
+		daemon, err := env.Cluster().Client().AppsV1().DaemonSets(kprobeGoCounterUserspaceNs).Get(ctx, kprobeGoCounterUserspaceDsName, metav1.GetOptions{})
+		require.NoError(t, err)
+		return daemon.Status.DesiredNumberScheduled == daemon.Status.NumberAvailable
+	}, time.Minute, time.Second)
```

Comment:
```suggestion
	},
	// Wait 5 minutes since cosign is slow, https://github.com/bpfman/bpfman/issues/1043
	5 * time.Minute, 10 * time.Second)
```
Add Andrew's temp work around.

Reply:
Is that still needed?  These have passed a few times without it.

Reply:
I would say just add it to keep them all consistent. If everything works, then no impact. If the network issue comes back, then all the tests will be covered.

Reply:
I copied this code before the workaround was added.  I agree with keeping them consistent.  However, 5 minutes seems like a long time.  I can add it to keep them consistent, but we should think about whether we want to keep the long wait.

Reply:
+1